assets/js/components/ClustersList.jsx

```diff
@@ -71,7 +82,26 @@ function ClustersList() {
         filterFromParams: true,
         filter: (filter, key) => (element) =>
           element[key].some((sid) => filter.includes(sid)),
-        render: (_, { sid }) => sid.join(', '),
+        render: (_, { sid, id }) =>
+          sid.map((singleSid, index) => {
```

Comment:
I personally prefer the way it is done in the `HostsList.jsx` component, in the `SID` column, but well, they both do the same at the end.

Reply:
No strong opinion here but consistency is nice :-)